### PR TITLE
Update according to Contributors library guidelines

### PR DESCRIPTION
**Pull request by milesfrain**
Initial migration according to the [Library Guidelines](https://github.com/purescript-contrib/governance/blob/main/library-guidelines.md).

Still some todos, including creating new issues for missing pieces.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,43 @@
-# purescript-ace
+# Ace
 
-[![Latest release](http://img.shields.io/github/release/slamdata/purescript-ace.svg)](https://github.com/slamdata/purescript-ace/releases)
-[![Build status](https://travis-ci.org/slamdata/purescript-ace.svg?branch=master)](https://travis-ci.org/slamdata/purescript-ace)
-[![Dependency status](https://img.shields.io/librariesio/github/slamdata/purescript-ace.svg)](https://libraries.io/github/slamdata/purescript-ace)
+[![CI](https://github.com/purescript-contrib/purescript-ace/workflows/CI/badge.svg?branch=main)](https://github.com/purescript-contrib/purescript-ace/actions?query=workflow%3ACI+branch%3Amain)
+[![Release](http://img.shields.io/github/release/purescript-contrib/purescript-ace.svg)](https://github.com/purescript-contrib/purescript-ace/releases)
+[![Pursuit](http://pursuit.purescript.org/packages/purescript-ace/badge)](http://pursuit.purescript.org/packages/purescript-ace)
+[![Maintainer: milesfrain](https://img.shields.io/badge/maintainer-milesfrain-teal.svg)](http://github.com/milesfrain)
```
Comment (milesfrain) [inline README.md]: Who wants to be the maintainer?

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+* PureScript record types are used for JavaScript objects.
+* JavaScript's fake sum types are mapped to PureScript's real sum types, which necessitates encoding / decoding.
+
+## Examples
+
+The `examples` directory contains an `index.html`, which contains a variety of examples of how to embed the Ace editor and configure it using the PureScript API.
+
+## Tests
+
+This project has no real tests. However, most bound functions are exercised in the examples, to ensure the bindings and necessary wrapping, unwrapping, encoding, and decoding has been performed correctly.
```
Comment (milesfrain) [inline docs/README.md]: Should these be split-up and moved to separate locations?

File: README.md (hunk 1 above)
Comment (thomashoneyman) [reply README.md]: If there is no maintainer assigned, then by default you can assign it to Gary and myself as we're responsible for these libraries in general. However, later this year we'd like to do a more formal process to identify other maintainers for these libraries.

File: docs/README.md (hunk 2 above)
Comment (thomashoneyman) [reply docs/README.md:20]: I think that these can be condensed and put in the `Documentation` section of the main README -- the section where it states where you can find different forms of documentation. I suggest:

1. Leave the API section as-is (though it ought to be improved over time)
2. Remove the Examples section here and add it as a new bullet point to the documentation section of the repository README (for example: `* The [examples directory](./examples) contains an HTML file demonstrating how to embed the Ace editor and configure it via PureScript`)
3. Remove the Tests section (users of the library don't really need to know this)

File: README.md [hunk 3]
```diff
@@ -1,28 +1,43 @@
...
-## Tests
+## Documentation
 
-This project has no real tests. However, most bound functions are exercised in the examples, to ensure the bindings and necessary wrapping, unwrapping, encoding, and decoding has been performed correctly.
+`ace` documentation is stored in a few places:
 
-## Documentation
+1. Module documentation is [published on Pursuit](https://pursuit.purescript.org/packages/purescript-ace).
+2. Written documentation and [the changelog](./docs/CHANGELOG.md) are kept in [the docs directory](./docs).
+3. Usage examples can be found in [the test suite](./test).
```
Comment (thomashoneyman) [inline README.md]: (this is where those sections from docs/README can be moved)

File: package.json [hunk 4]
```diff
@@ -1,15 +1,13 @@
...
   "devDependencies": {
-    "eslint": "^4.19.1",
-    "pulp": "^12.2.0",
-    "purescript": "slamdata/node-purescript#0.12",
-    "purescript-psa": "^0.6.0",
-    "rimraf": "^2.6.2"
+    "eslint": "^7.6.0"
+  },
+  "dependencies": {
+    "ace-builds": "^1.4.12"
```
Comment (thomashoneyman) [inline package.json:11]: I see this has moved from the `bower.json` file. That's good -- moving from Bower requires that and good catch! However, I believe that means the `index.html` file in the `examples` directory will also need to be updated to point at the correct location. Also, the repo README should be updated to note that you also need to install `ace-builds` to get the editor. For example:

> You will also need to install the Ace editor. For example, to install it from NPM:
> 
> ```sh
> npm install --save ace-builds
> ```

File: test/Main.purs [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+import Prelude
+
+import Effect (Effect)
+import Effect.Class.Console (log)
+
+main :: Effect Unit
+main = do
+  log "🍝"
+  log "You should add some tests."
```
Comment (thomashoneyman) [inline test/Main.purs:11]: I'm not sure what to do about the tests. Ideally we could run the editor, manipulate it, and verify the result is what we expect. But that might require something more involved, like a Puppeteer test, which I'm not particularly interested in maintaining.

File: docs/README.md (hunk 2 above)
Comment (thomashoneyman) [reply docs/README.md]: I think we can leave the API section as-is, remove the Examples section in favor of a simple bullet point in the main repo README (`* The [examples directory](./examples) demonstrates how to embed the Ace editor and configure it via the PureScript API`), and remove the tests section.

File: README.md (hunk 3 above)
Comment (thomashoneyman) [reply README.md]: We'd essentially adjust the third bullet to point to the examples instead of the test suite, as that doesn't have any usage examples.

**Comment by thomashoneyman** [comment]
You'll still need to delete the `.travis.yml` file, by the way.

File: package.json (hunk 4 above)
Comment (milesfrain) [reply package.json:11]: `--save` option is [no-longer required](https://stackoverflow.com/a/19578808).

**Comment by milesfrain** [comment]
Thanks for the thorough review feedback.
I applied all those changes. I believe this needs to be merged before opening up ToDo issues.

**Review by thomashoneyman — APPROVED** [review]

**Pull request merged**